versioned_docs/version-2.0.x/reference/global-options.mdx

```diff
@@ -559,7 +563,7 @@ Directory to use for named global caches for tools and processes with trusted, c
 <Option
   cli_repr={`--local-execution-root-dir=<str>`}
   env_repr='PANTS_LOCAL_EXECUTION_ROOT_DIR'
-  default_repr={`/tmp`}
+  default_repr={`<buildroot>`}
```

Comment:
Is this change correct?

Reply:
That one should be `<tmpdir>`: https://www.pantsbuild.org/docs/reference-global#local_execution_root_dir

Reply:
Ah yeah, this looks like a bug in the generations script, at some point I think `pants_workdir` changed defaults.

Reply:
I think we can let this one go, it looks like a "problem" in existing docs before 2.13:

- 2.12: https://www.pantsbuild.org/v2.12/docs/reference-global#local_execution_root_dir
![image](https://github.com/pantsbuild/pantsbuild.org/assets/1203825/5dbd4ad2-6d2e-4ad7-af87-b1600b2af74b)

- 2.13: https://www.pantsbuild.org/v2.13/docs/reference-global#local_execution_root_dir
![image](https://github.com/pantsbuild/pantsbuild.org/assets/1203825/24f5d59f-fd51-4a1f-ad8c-24450af7df65)



And indeed, the docs match, e.g. 2.20: https://github.com/pantsbuild/pantsbuild.org/blob/09d492d7962bf5535dad14ef8d28a46f14d72f45/docs/reference/global-options.mdx#L695-L696

Reply:
I split this into a dedicated issue: https://github.com/pantsbuild/pantsbuild.org/issues/60